charts/sn-platform/templates/bookkeeper/bookkeeper-cluster.yaml

```diff
@@ -107,6 +107,8 @@ spec:
         - {{ .Values.bookkeeper.configData.PULSAR_MEM }}
       gcOptions:
         - {{ .Values.bookkeeper.configData.PULSAR_GC }}
+      extraOptions:
```

Comment:
@fantapsody is this an example of the situation we talked about, that the operator applies various default values only when `extraOptions` is `[]`.  If so, then this change might cause other options to change, because they don't be applied.

Reply:
@EronWright seems so, which is not reasonable IMO, user provided ones should able to override whatever existing config, I think we should update the operator to put user provided arg at the end of jvm args as seems the right-most arg always wins
for this change I think we can just explicitly add whatever default jvm args operator will apply so we'll still have them, WDYT

Reply:
Yes please do that.

Reply:
updated, PTAL